docs/getting-started/create-block/attributes.md

```diff
@@ -41,6 +41,12 @@ The component is added similar to an HTML tag, setting a label, the `value` is s
 
 The save function will simply write the `attributes.message` as a div tag since that is how we defined it to be parsed.
 
+Install the components module in order to be able to use the TextControl component:
```

Comment:
This is not technically accurate as the DependencyExtractionPlugin that is part of the build process converts any imports from `@wordress/*` ( and some others ) to access the `wp` global - i.e `wp.components.TextControl`

Perhaps we can change this messaging to say that this is optional ( and why ) but is a good idea for IDE code completetion/code hinting?

Reply:
Interesting! I didn't even know about it. But my feeling as a developer following the tutorial was "Some info is missing here". See in this (#36550) issue the kind of problems one can have trying to follow the tutorial without this kind of info.

Could you please propose the changes in the text? Would something like the following be more accurate?

> Install the components module in order to be able to use the TextControl component (You don't actually need to do this because the build process would resolve the TextControl for you but it may be useful to install the dependency module to avoid issues with IDE code completion/code hinting):

Reply:
How does this text sound?

> OPTIONAL: For IDE support (code completion and hints), you can install the `@wordpress/components` module which is where the TextControl component is imported from. This install command is optional since the build process automatically detects @wordpress/* imports and specifies as dependencies in the assets file.

Reply:
> How does this text sound?

I think it sounds good. Let me replace it so we can see if the hole thing is ok or if it needs another adjustment.

Reply:
LGTM! 👍